torcharrow/functional.py

```diff
@@ -141,6 +141,60 @@ def add_tokens(
     return _dispatch("add_tokens", input_col, tokens, begin)
 
 
+def bpe_tokenize(
+    tokenizer,
```

Comment:
@wenleix I'm wondering what the correct type would be for the `tokenizer` param here